### PR TITLE
Feat/return event datepicker

### PR DESCRIPTION
**Pull request by SercanSercan**
## 📥 Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: packages/datepicker-react/src/DatePicker.tsx [hunk 1]
```diff
@@ -138,37 +140,37 @@ export function DatePicker({
     const handleBlur = (e: FocusEvent) => {
         const nextFocusIsInside = componentRef.current && componentRef.current.contains(e.relatedTarget as Node);
         if (onBlur && !nextFocusIsInside) {
-            onBlur(date);
+            onBlur(date, e);
```
Comment (lfbergee) [inline packages/datepicker-react/src/DatePicker.tsx:143]: Kanskje rydde opp inkonsistens mellom funksjonene her, velg enten e eller event, ikke begge.. ikke din feil, men greit å få rydda.

File: packages/datepicker-react/README.md [hunk 2]
```diff
@@ -62,3 +62,14 @@ Du kan sette en forhåndsvalgt dato med `initialDate`. Dersom ingen dato er valg
 ```jsx
 <DatePicker initialDate={new Date(new Date().setFullYear(new Date().getFullYear() + 1))} />
 ```
+
+## Event Handling
+DatePicker komponent mottar 3 event funksjoner (onFocus, onChange og onBlur). 
+Disse eventer returnerer en native event object og en Date object. Både parameterer er optional.
```
Comment (piofinn) [inline packages/datepicker-react/README.md:68]: Litt språkvask, siden du ba meg om det 😊

```suggestion
DatePicker-komponenten mottar tre event-funksjoner (onFocus, onChange og onBlur). 
Disse eventene returnerer et native event-objekt og et Date-objekt. Begge parametrene er optional. 
```

**Review by piofinn — DISMISSED** [review]

**Comment by piofinn** [comment]
Approved som den er, men ta gjerne inn forslaget til @lfbergee og språkvasken min hvis du gidder 😊

File: packages/datepicker-react/src/DatePicker.tsx (hunk 1 above)
Comment (piofinn) [reply packages/datepicker-react/src/DatePicker.tsx:143]: Det er nok min feil 😅 Sikkert fint å få rydda når man uansett gjør endringer ja.

Comment (SercanSercan) [reply packages/datepicker-react/src/DatePicker.tsx:143]: Takk for forslaget.

File: packages/datepicker-react/README.md (hunk 2 above)
Comment (SercanSercan) [reply packages/datepicker-react/README.md]: Tusen takk! Ble endret.

**Review by piofinn — APPROVED** [review]

**Pull request merged**